x-pack/plugins/security_solution/public/flyout/document_details/right/footer.tsx

```diff
@@ -61,12 +68,12 @@ export const PanelFooter: FC = () => {
         detailsEcsData={dataAsNestedObject}
         handleOnEventClosed={closeFlyout}
         isHostIsolationPanelOpen={isHostIsolationPanelOpen}
-        isReadOnly={false}
+        isReadOnly={isPreview}
```

Comment:
this change isn't necessary as `isPreview` will always be false here, but I guess the change can't hurt either...